src/pages/Doyun/Main/components/Feed/Comment.scss

```
@@ -0,0 +1,13 @@
+div.comment {
```

```suggestion
.comment {
```
필요한 셀렉터까지만 적어주셔도 될거 같습니다